zowe/vscode-extension-for-zowe
# Fix issue 1144: Persist removed favorite profiles between sessions

## Proposed changes
Fixes #1144 (Remove favorite Profile from all the trees in THEIA and VScode not working).

To test this PR:
1. Add profile to Favorites (Under any tree Datasets/USS/Jobs).
2. Go to Favorites.
3. Right click on favorite profile and select "Remove profile from Favorites" and select "Continue".
4. See that the profile is removed from the Favorites section.
5. Refresh VS Code (or Theia), and check that the profile has not reappeared in the Favorites section.

## Release Notes

Milestone: 1.12.1, if possible.

Changelog: Fixed persisting removed favorite profiles between IDE sessions.

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_


## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_
